features/fork.feature

```diff
@@ -34,9 +34,12 @@ Feature: hub fork
         json :name => 'dotfiles', :owner => { :login => 'mislav' }
       }
       """
-    And I successfully run `git remote rename origin upstream`
     When I successfully run `hub fork --remote-name=origin`
-    Then the output should contain exactly "new remote: origin\n"
+    Then the output should contain exactly:
+      """
+      renaming existing "origin" remote to "upstream"
```

Comment:
This verifies command ouput, but can there be an additional step that verifies that the `upstream` remote now exist and that it points to the same thing `origin` used to point to?

Reply:
would
```
diff --git a/features/fork.feature b/features/fork.feature
index f5738cb..0afcb18 100644
--- a/features/fork.feature
+++ b/features/fork.feature
@@ -43,6 +43,7 @@ Feature: hub fork
     And "git remote add -f origin git://github.com/evilchelu/dotfiles.git" should be run
     And "git remote set-url origin git@github.com:mislav/dotfiles.git" should be run
     And the url for "origin" should be "git@github.com:mislav/dotfiles.git"
+    And the url for "upstream" should be "git@github.com:evilchelu/dotfiles.git"
 
   Scenario: Fork the repository with redirect
     Given the GitHub API server:
```
be enough?

Reply:
Yep!